### ISSUE TITLE
Run websearch selenium plugin

### ISSUE THREAD

**Issue by kevinleijh** [SillyTavern/SillyTavern-WebSearch-Selenium#2]
How to run the websearch selenium plugin? can you clarify the instructions? thank you

**Comment by kevinleijh** [comment]
Thank you what do you mean by configuration? When I try to search nothing happens. Will the plugin work with windows?

**Comment by mpetruc** [comment]
@kevinleijh have you been to figure out how to use the webserch+selenium ? I'm also running ST in windows. Just to make it clear,
- i installed the selenium plugin. It is located inside the SillyTavern folder, in a folder called SillyTavern-WebSearch-Selenium.
- set SILLYTAVERN_SELENIUM_BROWSER=firefox
- modified the config.yaml file with enableServerPlugins: true
- installed and enabled the websearch plugin in ST and enabled it. Selected Source=Selenium plugin and Engine=DuckDuckGo
- When prompting with "find me..." as trigger nothing happens.

thanks!

**Comment by Neuro-Storm** [comment]
@mpetruc @kevinleijh Did it work for anyone? Do you know any other plugins for web search?

**Comment by mpetruc** [comment]
@Neuro-Storm no. Quite unfortunate. I hope if anyone does to at least confirm it's doable. Don't have any intact walls in the house to bang my head against trying to figure out how to make it work. It's probably something stupidly simple that i'm overlooking.

**Comment by The-unknown-Shadowman** [comment]
Under "API connection" tab "Chat-Completions" has to be active and server has to be configured.
Then under "AI response configuration" tab you can activate "enable function calling".
The folder "SillyTavern-Websearch-Selenium" has to be in "plugins" folder inside SillyTaver folder.
